python_tests/test_python_frontend.py

```diff
@@ -1382,6 +1383,27 @@ def test_compute_contiguity(self):
         contiguity = [False, None, True, None, None, True, False]
         self.assertEqual(compute_contiguity(sizes, strides), contiguity)
 
+    def test_compute_tensor_descriptor(self):
+        sizes = [2, 1, 3, 1, 4, 3]
+        strides = [12, 4, 4, 4, 1, 0]
+        contiguity = [True, None, True, None, True, None]
+        stride_order = [5, 4, 3, 2, 1, 0]
```

Comment:
I'm trying to understand this example. Is stride order uniquely determined by the sizes and strides when expanded dims are present? In this case wouldn't [5, 4, 2, 1, 3, 0] be a valid stride order also? i.e. any sort of the strides. Do we guarantee a particular order in the cases?

Reply:
Oops. sorry for the lack of comment. I was trying to confirm I'm returning the right order before writing down any comment/doc.

Reply:
In short, there's not a unique ordering when we have equal strides.

But we do preserve the original order and rely on that to break tie.